spacksites/spack-env-templates/slurm/youngmichael.yaml

```diff
@@ -0,0 +1,23 @@
+spack:
+
+  specs:
+  - matrix:
+    - - orca@6.0.1
+    - - '%gcc@12.3.0'
+  concretizer:
+    unify: when_possible
+  view: false
+  include_concrete:
+  - /lustre/apps/spack/0.23/deploy/2025-02/spack/var/spack/environments/avx2
+  - /lustre/apps/spack/0.23/deploy/2025-02/spack/var/spack/environments/avx2_buildtools
+  - /lustre/apps/spack/0.23/deploy/2025-02/spack/var/spack/environments/avx512
+  - /lustre/apps/spack/0.23/deploy/2025-02/spack/var/spack/environments/multiarch
```

Comment:
I don't think these paths exist yet (I saw that on Michael there's `/lustre/shared/ucl/apps/spack/0.23/`, but not a deployed system), so these are placeholders for the time being.